packages/material-ui/src/Slider/Slider.js

```diff
@@ -247,11 +247,12 @@ export const styles = theme => ({
     '&::after': {
       position: 'absolute',
       content: '""',
-      // reach 48px touch target (2 * 18 + thumb circumference)
-      left: -18,
-      top: -18,
-      right: -18,
-      bottom: -18,
+      borderRadius: '50%',
+      // reach 42px hit target (2 * 15 + thumb diameter)
```

Comment:
Where is the 42px coming from?

Reply:
It's a tradeoff to be closer to 44px without leaking too much from the dimension of the parent. 28px is a tradeoff to increase the interaction zone without changing too much the layout for existing users and still enabling dense layout.

I don't think that we should follow the specification to the later.

Reply:
But if you don't you have to explain how you arrive at these values. And not just some ephemeral "it's a tradeoff". Given the 8px grid this is now 3/4 of a unit instead of the 1/2 which is probably why the recommendation uses 44. Just  go with the default to stay in the design system. Not following the default system is actually more likely to cause layout breakage.

Reply:
I think that we can ignore the 4px grid in this case because the element is not visible. By tradeoff, I mean, increase the slider height by the minimal unit: 4px, make the thumb "leak" by the minimum amount: 7px visually look almost too much.

Reply:
What is this "minimum amount" you talk about? How did you construct this number?

Reply:
4px is the grid unit